src/controllers/userController.js

```diff
@@ -39,6 +39,31 @@ class UserControl {
     }
   }
 
+  static async getAllByEmail(req, res, next) {
+    const {
+      email,
+    } = req.params;
+
+    if (email == req.decoded.email) {
```

Comment:
Remove email from token

Reply:
These are the things I will be working on the feedback